cmdx.py

```diff
@@ -1246,10 +1252,16 @@ def __repr__(self):
     def __init__(self, mobject, *args, **kwargs):
         super(DagNode, self).__init__(mobject, *args, **kwargs)
 
-        # Convert self._tfn to om.MFnTransform(self.dagPath())
-        # if you want to use its functions which require sWorld
         self._tfn = om.MFnTransform(mobject)
 
+        # additional cache placeholders
+        self._mdagpath = None
+
+        # Create self._dagtfn with om.MFnTransform(self.dagPath())
+        # if you want to use dag node functions which require sWorld
```

Comment:
- Lowercase sentence, `Additional...`
- Typo "Create self._dagfn..."

Reply:
i have to pay attention to lowercase. I'm used to lowercase everthing. bad habit etc.. (:

and for self._dagtfn it was not a typo but i'll rewrite some additional explanations for these. Maybe i'll revise these cache from what you noticed with dagpath cache